locale/misc/sv.yaml

```diff
@@ -1,4 +1,5 @@
 callLog:
+    note: "Ingen anteckning lämnad"
```

Comment:
Is there any particular reason you decided to not mention the caller as suggested in #73?